examples/ros1-chatter/package.json

```diff
@@ -1,18 +1,18 @@
 {
   "name": "ros1-chatter",
   "version": "0.0.0",
-  "description": "Example @foxglove/ros1 client that publishes to /chatter",
+  "description": "Example @lichtblick/ros1 client that publishes to /chatter",
   "license": "MIT",
   "private": true,
   "repository": {
     "type": "git",
-    "url": "https://github.com/foxglove/ros1.git"
+    "url": "https://github.com/Lichtblick-Suite/ros1"
```

Comment:
```suggestion
    "url": "https://github.com/Lichtblick-Suite/ros1.git"
```

Reply:
I have been removing .git on all urls because in some cases we had problems publishing the packages on NPM, but I can add it back